errr/exception.py

```diff
@@ -58,6 +58,11 @@ def __getattr__(self, attr):
             return self.details[attr]
         return self.__getattribute__(attr)
 
+    def __reduce__(self):
+        # Need to define it for tblib>=3.2 to work
+        # despite it doing nothing more than calling the parent function
+        return super().__reduce__()
```

Comment:
Are you sure we don't need to implement something here? I would've guessed that the reduce function needs to return some sort of set of arguments to (de)ser this instance. Can you add some multiprocessing tests that show it can be transmitted and reconstructed on the other side correctly? You can look at `mpipool` source code to see how exceptions are sent across ranks.